help/marking-messages-as-read.md

```diff
@@ -23,6 +23,27 @@ configure whether messages are marked as read automatically when you scroll.
 
 {end_tabs}
 
+## Do not mark messages as read
+
+Some users may want to configure the Zulip mobile app to never mark messages as
+read automatically as part of their workflow. This can be useful if you only
+want to glance at your new messages from the mobile app and then read them later
+on the desktop or web app.
```

Comment:
Maybe something like this for the intro?

> You may want to configure the Zulip mobile app to never mark messages as
read automatically. For example, this can be useful if you often want to follow up on messages when you are at your computer.